apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -304,12 +322,14 @@ private class HistoryEntry
                     }
                 }
             }
+            LogPerformanceDataAfterTest();
         }
 
         [Fact]
         public async Task ReadExactSingle()
         {
             await WriteSampleRowsAsync();
+            LogPerformanceDataBeforeTest();
             await Task.Delay(6);
```

Comment:
Should this delay be logged as part of the test? Seems like the 'before' logging should be done after this delay.